rollbar/__init__.py

```diff
@@ -807,46 +798,28 @@ def _add_locals_data(data, exc_info):
             #       return x + a + b + z
             #
             #   func((1, (1, 2), 3))
-            named_args = _flatten_nested_lists(arginfo.args)
-
-            # Fill in all of the named args
-            for named_arg in named_args:
-                if named_arg in local_vars:
-                    args.append(local_vars[named_arg])
+            args.extend(_flatten_nested_lists(arginfo.args))
 
-            # Add any varargs
-            if arginfo.varargs is not None:
-                args.extend(local_vars[arginfo.varargs])
+            # Remove varargs if desired
+            if arginfo.varargs is not None and SETTINGS['locals']['scrub_varargs']:
```

Comment:
I think we still want to include varargs in the argspec if we don't delete it here. 

E.g. in rollbar.com UI I would expect to see something like:

## If the notifier were configured to not scrub varargs:
```py
def foo(named_arg_1, named_arg_2, *extra_args):
    pass

# called with
foo('hello', 'cruel', 'world', '!')
```
Would show this in the rollbar.com UI

```
+--------------------------------------------
|  args
+--------------------------------------------
|  named_arg_1   |    "hello"
|  named_arg_2   |    "cruel"
|  extra_args    |    ["world", "!"]
+--------------------------------------------
```

## If the notifier were not configured to not scrub varargs I would expect to see:

```
+--------------------------------------------
|  args
+--------------------------------------------
|  named_arg_1   |    "hello"
|  named_arg_2   |    "cruel"
+--------------------------------------------
```